deployability/modules/workflow_engine/workflow_processor.py

```diff
@@ -33,9 +34,14 @@ def __validate_schema(self, workflow_file: Path | str) -> None:
         Args:
             workflow_file (Path | str): Path to the workflow file.
         """
-        validator = SchemaValidator(self.schema_path, workflow_file)
-        validator.preprocess_data()
-        validator.validateSchema()
+        try:
+            logger.debug(f"Validating inptu file: {workflow_file}")
```

Comment:
Typo: inp`tu` -> inp`ut`

Reply:
Done in [ebce158](https://github.com/wazuh/wazuh-qa/pull/4960/commits/ebce158066ecff2dc0cb82cc7f902dc54b6d489d)